internal/corazawaf/transaction.go

```diff
@@ -746,6 +746,14 @@ func (tx *Transaction) ProcessURI(uri string, method string, httpVersion string)
 	tx.variables.queryString.Set(query)
 }
 
+// ProcessServerName allows to set server name details.
+//
+// The API consumer is in charge of retrieving the value (e.g. from the host header).
+// It is expected to be executed before calling ProcessRequestHeaders.
```

Comment:
I'd call it `SetServerName`, `Set` because the method should represent the action and drop `Process` because I don't see how this need the process word.

Also, `It is expected to be executed before calling ProcessRequestHeaders.` this should be explicit in the code, maybe dropping a log line when this is not the case (e.g. `[WARN] setServerName called before ProcessRequestHeaders).

Reply:
It was named `Process` for name consistency, but I agree that it can be misleading because no processing is effectively done